Oracle/run-i2p-transform.sh

```diff
@@ -16,6 +16,8 @@ set -e
 #export network_id=
 #export network_name=
 # export i2b2_etl_schema=
+# export min_pat_list_date_dd_mon_rrrr=
```

Comment:
Maintaining this list is getting boring. But I don't suppose shell scripts are self-documenting w.r.t. this sort of interface stuff.

Actually, we could move this comment to code if we used a shell function. Worth the trouble?